src/it/java/com/google/checkstyle/test/base/AbstractModuleTestSupport.java

```diff
@@ -133,7 +133,6 @@ public Checker createChecker(Configuration moduleConfig)
      * @param moduleConfig {@link Configuration} instance.
      * @return {@link Checker} instance.
      * @throws CheckstyleException if an exception occurs during checker configuration.
-     * @noinspection BooleanParameter
```

Comment:
Why is it redundant when we were required to make this change in test?
I even made a comment about this in https://github.com/checkstyle/checkstyle/issues/4857#issuecomment-320938777 .